.github/CHANGELOG.md

```diff
@@ -75,6 +75,7 @@
 1. [LIGHTS] Added logo lights only turn on while on ground or flaps are extented - @linkeleven (linkeleven#2557)
 1. [CDU] Add basic airways support - @santii90 (Santiago Vazquez), @Lessar277 (Lessar27)
```

Comment:
missing credits for code donation from WorkingTitle

Reply:
Sorry, I was going to ensure this was done but forgot.